mongo/datadog_checks/mongo/mongo.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, name, init_config, instances=None):
         # Members' last replica set states
         self._last_state_by_server = {}
 
-        self.collection_metrics_names = (key.split('.')[1] for key in metrics.COLLECTION_METRICS)
+        self.collection_metrics_names = tuple([key.split('.')[1] for key in metrics.COLLECTION_METRICS])
```

Comment:
```suggestion
        self.collection_metrics_names = tuple(key.split('.')[1] for key in metrics.COLLECTION_METRICS)
```

Do you need the extra case?

Reply:
Apprently it's better/faster to do that in python 🤷